Add unit tests to new Jenkins (#70)
CanalTP/SamCoreBundle: README.md

- Entities (Application, Customer, Role...)
- Customer management

Unit Tests
----------

> <b>Requirements : [docker](https://docs.docker.com/get-docker/)</b>
> 
> You could launch the unit tests with this steps :

Reviewer: ```suggestion
**Requirements** : [docker](https://docs.docker.com/get-docker/)

You can launch unit tests following those steps :
```

1. Build docker image
```
mkdir -p ${HOME}/.config/composer
_UID=$(id -u) GID=$(id -g) docker-compose -f docker-compose.test.yml build --no-cache --force-rm --pull samcore-app
```
2. Launch composer
```
rm -f composer.lock
_UID=$(id -u) GID=$(id -g) docker-compose -f docker-compose.test.yml run --rm samcore-app composer install --no-interaction --prefer-dist
```

> 3. Launch unit tests

Reviewer: ```suggestion
Note : first launch can be very long (>45mins) depending on your device resources.

3. Launch unit tests
```
